client/features/checkout/Review.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { withStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemText from "@material-ui/core/ListItemText";
+import Grid from "@material-ui/core/Grid";
+import { usdFormatter, getShippingCost } from "../../helpers";
+
+const addresses = [
+  "1 Material-UI Drive",
+  "Reactville",
+  "Anytown",
+  "99999",
+  "USA",
+];
+const payments = [
+  { name: "Card type", detail: "Visa" },
+  { name: "Card holder", detail: "Mr John Smith" },
+  { name: "Card number", detail: "xxxx-xxxx-xxxx-1234" },
+  { name: "Expiry date", detail: "04/2024" },
+];
+
+const styles = (theme) => ({
+  listItem: {
+    padding: `${theme.spacing.unit}px 0`,
+  },
+  total: {
+    fontWeight: "700",
+  },
+  title: {
+    marginTop: theme.spacing.unit * 2,
+  },
+});
+
+function Review(props) {
+  const { classes } = props;
+  return (
+    <React.Fragment>
+      <Typography variant="h6" gutterBottom>
+        Order summary
+      </Typography>
+      <Grid container spacing={16}>
+        <Grid item xs={12} sm={6}>
+          <Typography variant="h6" gutterBottom className={classes.title}>
+            Shipping
+          </Typography>
+          <Typography gutterBottom>John Smith</Typography>
```

Comment:
John Smith ? I think the data we enter into the form should be displaying here right ?